Dynavity/Dynavity/view-model/canvas/CanvasViewModel.swift

```diff
@@ -6,6 +6,18 @@ class CanvasViewModel: ObservableObject {
     @Published var annotationCanvas = AnnotationCanvas()
     @Published var selectedImage: UIImage?
 
+    // Selected media
+    @Published var selectedImage: UIImage? {
```

Comment:
There's a duplicate declaration in line 7. Is this intentional?

Reply:
Thanks for spotting this. It creeped in after merging `master` into the branch.